realm/realm-library/src/main/java/io/realm/internal/modules/CompositeMediator.java

```diff
@@ -115,6 +115,14 @@ public String getTableName(Class<? extends RealmObject> clazz) {
         return mediator.createDetachedCopy(realmObject, maxDepth, cache);
     }
 
+    @Override
+    public boolean transformerApplied() {
+        if (mediators.isEmpty()) {
```

Comment:
Right, this is not accurate. But there is no way to detect whether transformed or not if `mediators` is empty.

Now we only checks the return value of `DefaultRealmModuleMediator` class in `RealmConfiguration`.
That means the implementation of `CompositeMediator` and `FilterableMediator`
does not affect the result of the check.

Is returning `true` better than current code?